<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Control</string>
  </property>
  <property name="windowIcon">
   <iconset theme="multimedia-volume-control">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="notebook">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Playback</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="streamsVBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>730</width>
            <height>423</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="noStreamsLabel">
             <property name="text">
              <string>&lt;i&gt;No application is currently playing audio.&lt;/i&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Show:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="sinkInputTypeComboBox">
         <item>
          <property name="text">
           <string>All Streams</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Virtual Streams</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;Recording</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="recsVBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>730</width>
            <height>423</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="noRecsLabel">
             <property name="text">
              <string>&lt;i&gt;No application is currently recording audio.&lt;/i&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Show:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="sourceOutputTypeComboBox">
         <item>
          <property name="text">
           <string>All Streams</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Applications</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Virtual Streams</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>&amp;Output Devices</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="sinksVBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>730</width>
            <height>423</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="noSinksLabel">
             <property name="text">
              <string>&lt;i&gt;No output devices available&lt;/i&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Show:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="sinkTypeComboBox">
         <item>
          <property name="text">
           <string>All Output Devices</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hardware Output Devices</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Virtual Output Devices</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>&amp;Input Devices</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollArea_4">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="sourcesVBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>730</width>
            <height>423</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="noSourcesLabel">
             <property name="text">
              <string>&lt;i&gt;No input devices available&lt;/i&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Show:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="sourceTypeComboBox">
         <item>
          <property name="text">
           <string>All Input Devices</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All Except Monitors</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hardware Input Devices</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Virtual Input Devices</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Monitors</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>&amp;Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QScrollArea" name="scrollArea_5">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="cardsVBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>730</width>
            <height>433</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="noCardsLabel">
             <property name="text">
              <string>&lt;i&gt;No cards available for configuration&lt;/i&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showVolumeMetersCheckButton">
         <property name="text">
          <string>Show volume meters</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="connectingLabel">
     <property name="text">
      <string>...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
